apps/src/lib/levelbuilder/code-docs-editor/ProgrammingExpressionEditor.jsx

```diff
@@ -257,7 +257,7 @@ export default function ProgrammingExpressionEditor({
         isSaving={isSaving}
         lastSaved={lastUpdated}
         error={error}
-        handleView={() => navigateToHref('/')}
+        handleView={() => navigateToHref(`/programming_expresions/${id}`)}
```

Comment:
I think there's a typo in "expresions" (one s instead of two) although if I stare at any word long enough, it starts to look right

Reply:
Other handleViews are using `navigateToHref(linkWithQueryParams('path')` ... do you want to add linkWithQueryParams here too?

Reply:
omg I though I fixed the typo but I guess I didn't...

And yeah, good call out using `linkWithQueryParams`. I'll update

Reply:
My question then is what the downside is to using `linkWithQueryParams`? Is there an unexpected behavior that will happen if query params are preserved?

Reply:
potential downsides:
1. unnecessary complexity 
2. query params could become cache-busters in other scenarios if this pattern is replicated indiscriminately. for example, when a teacher views a lesson plan, they may have a section id in the url. when they go to view the PDF of the lesson plan, that link definitely shouldn't pass url params along, or it could make following that link result in a cache miss.

Reply:
I'm thinking about making default handleView in SaveBar, partly to avoid the need to write tests for things we're doing again and again in each editor. Would you then suggest making default behavior to *not* use linkWithQueryParams?

Reply:
yes :-) until/unless we can think of a query param that we'd need to preserve.

Reply:
Excellent!